### PR TITLE
Anfield Enhancements: Specific Import & Storage domain Import Fix

### PR DESCRIPTION
**Pull request by naveena-maplelabs**
1) Option to provide prefix and suffix for imported protection jobs.
2) List of resources to be imported can be provided as json.
   i) If list is empty all the sources are imported, orelse only
      provided resources are imported.
   ii) resources include sources, policies, jobs, views , storage
       domains, targets and remote clusters.

### CODE REVIEW

File: samples/project-anfield/resources.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "jobs": [],
```
Comment (pyashish) [inline samples/project-anfield/resources.json:2]: As discussed today, how are you making sure that the user isn't getting into error if they are deleting policy / entity before the job ?

**Review by pyashish — CHANGES_REQUESTED** [review]

File: samples/project-anfield/import_cluster_config.py [hunk 2]
```diff
@@ -664,6 +696,14 @@ def create_protection_jobs():
             _parent_id = None
             is_job_available = False
             job_name = protection_job.name
+            if jobs_to_import and job_name not in jobs_to_import:
+                # Check job is available in the json, if no jobs is added all
+                # the jobs are protected, or else only selected jobs are
+                # protected.
+                continue
+            if job_prefix or job_suffix:
```
[the remark this answers is not in the document]
Comment (naveena-maplelabs) [reply samples/project-anfield/import_cluster_config.py:704]: Okay Ashish

File: samples/project-anfield/config.ini [hunk 3]
```diff
@@ -19,8 +19,10 @@ override=True
 # gflag parameter is for to decide whether to import gflags.
 gflag=False
 # prefix and suffix added to Jobname. By default no prefix/suffix is added.
-job_prefix=
-job_suffix=
+imported_job_prefix=
+imported_job_suffix=
+# List of jobs to be imported. Accepts list of comma separated strings.
```
Comment (pyashish) [inline samples/project-anfield/config.ini:24]: Good way to show example

File: samples/project-anfield/import_cluster_config.py [hunk 4]
```diff
@@ -982,18 +971,11 @@ def create_remote_clusters():
     for cluster in existing_cluster_list:
         repl_list[cluster.name] = cluster.cluster_id
 
-    remote_clusters_to_import = resources_to_import.get("remote_clusters", [])
     # If the remote cluster exists then get the ID.
     for cluster in remote_cluster_list:
         # Check the remote cluster registered is not the current cluster.
         if cluster.remote_ips[0] == import_cluster_ip:
             continue
-        if remote_clusters_to_import and cluster.remote_ips[0] not in \
```
Comment (pyashish) [inline samples/project-anfield/import_cluster_config.py]: Why is this removed ?

File: samples/project-anfield/import_cluster_config.py [hunk 5]
```diff
@@ -100,6 +100,9 @@
         print("Import file does not exist")
         sys.exit(1)
 
+with open("resources.json") as file_obj:
```
[the remark this answers is not in the document]
Comment (naveena-maplelabs) [reply samples/project-anfield/import_cluster_config.py]: Done

File: samples/project-anfield/resources.json (hunk 1 above)
Comment (naveena-maplelabs) [reply samples/project-anfield/resources.json]: Importing jobs is specific

File: samples/project-anfield/config.ini (hunk 3 above)
Comment (naveena-maplelabs) [reply samples/project-anfield/config.ini:24]: Can we add the example in documentation?

Comment (pyashish) [reply samples/project-anfield/config.ini:24]: Sure
